### PR TITLE
test(vulnerable-code): Add a test for a Commons-Compress CVE

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

**Comment by sschuberth** [comment]
See https://oss-review-toolkit.slack.com/archives/C9NNJ54B1/p1707389030189259 for context.

File: plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt [hunk 1]
```diff
@@ -30,27 +30,51 @@ import org.ossreviewtoolkit.model.Package
...
-                "CVE-2023-2976"
-            )
-
-            getValue("CVE-2023-2976").references.find {
-                it.url.toString() == "https://nvd.nist.gov/vuln/detail/CVE-2023-2976"
-            } shouldNotBeNull {
-                severity shouldBe "7.1"
-                severityRating shouldBe "HIGH"
+class VulnerableCodeFunTest : WordSpec({
+    "Vulnerable Maven packages" should {
```
Comment (fviernau) [inline plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt:34]: Why did you name this "Vulnerable Maven packages"? Have you considered `"retriveFindings()"` instead?

File: plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt [hunk 2]
```diff
@@ -30,27 +30,51 @@ import org.ossreviewtoolkit.model.Package
...
+        "return findings for Guava" {
+            val vc = VulnerableCode("VulnerableCode", VulnerableCodeConfiguration())
+            val id = Identifier("Maven:com.google.guava:guava:19.0")
+            val pkg = Package.EMPTY.copy(id, purl = id.toPurl())
+
+            val findings = vc.retrievePackageFindings(setOf(pkg))
```
Comment (fviernau) [inline plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt:40]: commit: Why do we need a test for another Maven package?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt (hunk 2 above)
Comment (sschuberth) [reply plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt:40]: Please see [my comment above](https://github.com/oss-review-toolkit/ort/pull/8256#issuecomment-1938161046).

File: plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt (hunk 1 above)
Comment (sschuberth) [reply plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt:34]: My idea was to pave the way to add tests for more ecosystems, like NPM, as well, and prepare the naming for that.

**Review by fviernau — APPROVED** [review]

**Pull request merged**